packages/client-web-kit/src/components/entities/identity-provider-role/AIdentityProviderRoleAssignment.vue

```
@@ -12,7 +12,7 @@ import { maxLength, minLength } from '@vuelidate/validators';
 import type { PropType } from 'vue';
 import { defineComponent, reactive, ref } from 'vue';
 import type { IdentityProviderRoleMapping, Role } from '@authup/core-kit';
-import { VCFormGroup, VCFormInput, VCFormInputCheckbox } from '@vuecs/form-controls';
+import { VCFormGroup, VCFormInput, VCFormCheckbox } from '@vuecs/forms';
```

ESLint

[error] Member 'VCFormCheckbox' of the import declaration should be sorted alphabetically.

(sort-imports)